docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+        version control, checked in, etc. and has the benefit of only
+        defining/changing the specific area required, while ensuring any
+        non-changing areas are left untouched. The rule here to keep in mind
+        where multiple yaml files are applied, the later ones will always
+        overwrite the values in the earlier ones. They are merged in in
+        sequence.</para>
+
+        <para>Another point to consider, where there is a global dictionary
+        such as root: and its value redefined in the 2nd file (as a
+        dictionary) it would not be overwritten. There is no “nice way” to
+        replace a dictionary, you cant overwrite a dictionary. You can
+        redefine a dictionary and set it to null, which will effectively wipe
+        it out.</para>
+
+        <para><emphasis role="bold">WARNING</emphasis>: If you had a global
+        definition (such as Storage: Planes:) and merge it where that becomes
+        redefined it would wipe out every definition in the list.</para>
+
+        <para>Another means to wipe out every value in a list is to pass in an
+        empty set denoted by a [ ] such as this example:</para>
+
+        <para><programlisting>bundles: []</programlisting>This would wipe out
+        any properties defined for bundles.</para>
+      </sect3>
+
+      <sect3 id="GenerallyApplicable">
+        <title>Generally applicable</title>
+
+        <para>These items are generally applicable for our HPCC Systems Helm
+        yaml files.</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>All names need to be unique.</para>
+          </listitem>
+
+          <listitem>
+            <para>All prefixes should be unique.</para>
+          </listitem>
+
+          <listitem>
+            <para>Services should be unique.</para>
+          </listitem>
+
+          <listitem>
+            <para>yaml files are merged in sequence.</para>
+          </listitem>
+        </itemizedlist>
+      </sect3>
+
+      <sect3 id="Global_Chart_Values">
+        <title>Global chart values</title>
+
+        <para>items defined in the global section are shared between all
+        components</para>
+
+        <para>Examples of a global values would be in delivered values.yaml
+        file the storage section and then the security section.</para>
+
+        <programlisting>storage:
+  planes:</programlisting>
+
+        <para>and also</para>
+
+        <para><programlisting>security:
+  eclSecurity:
+    # Possible values:
+    # allow - functionality is permitted
+    # deny - functionality is not permitted
+    # allowSigned - functionality permitted only if code signed
+    embedded: "allow"
+    pipe:  "allow"
+    extern: "allow"
+    datafile: "allow"</programlisting>In the above examples, storage: and
+        security: are global chart values.</para>
+      </sect3>
+    </sect2>
+
+    <sect2>
+      <title>Adding Components</title>
```

Comment:
This section doesn't really seem to be describing how to add a component - or even which part of the values file relates to the components.  This could either be in with the description of the shipped values file, or this section could be expanded.  E.g., the components are found in this section, each is a list of instances, they share some characteristics such as...